testing-endpoints.md

```diff
@@ -18,20 +18,60 @@ curl -X GET http://localhost:8080/api/test/did-debug
 ```
 
 
+#### `api/test/generate-did`
+
+Returns a Long-Form did made from input public keys in jwk format
+
+The body is a JSON object (note, this API requires a content-type header) with the following fields:
+1. `recoveryPublicJwk` - **required** `ES256` public signing key jwk for did recovery operations
+2. `updatePublicJwk` - **required** `ES256` public signing key jwk for did update operations
+3. `signingPublicJwk` - **optional** `ES256` public signing key jwk for verifying assertions 
+4. `encryptionPublicJwk` - **optional** public encryption key jwk using the `"alg": "ECDH-ES"` and `"enc": "A256GCM"` encryption algorithm
+5. `domains` - **optional** array of string URI hostnames. Will be added to the did's `LinkedDomains` by an `add-services` action. 
+
+**Note**: The `signingPublicJwk` and `encryptionPublicJwk` fields are optional since 
+not all entities represented by DIDs may need to sign or encrypt documents.
+
+For exmample, a `verifier` or `issuer` entity that does not request SIOP Responses to be encrypted MAY only need to have a signing key (for signing SIOP requests), and not an encryption key.
+
+Example key props for signing keys:
+```
+{
+    "kty": "EC",
+    "crv": "P-256",
+    "alg": 'ES256',
```

Comment:
include `"x":"rAkPWQoZai50iSgCkBQ-eHR9O4L7edymIAUB1m9O6c8","y":"H-ulzimHfP-LyJYpPeIUoNguNs7jFfqaCncE6ZKkLUc"` for completeness? Similar for encryption key prop below?

Reply:
I think I'd rather just remove these objects. I only put them in as reference to the JOSE library inputs, but I'd rather omit than try to be too rigorous here.

Reply:
Also, there are example JWKs in the request, and those have all the values!